### DISCUSSION TITLE
Can't find sddmwall.sh

### DISCUSSION THREAD

**Discussion by AfthabEK** [prasanthrangan/hyprdots#1133] [Q&A] [answered]
Can't find sddmwall.sh in  ~/.config/hypr/scripts/sddmwall.sh, there is no directory scripts in ~/.config/hypr, so I can't change my lock screen wallpaper.

**Comment by kRHYME7** [comment]
the directory is move to ~/.config/hyprdots/ 
but the sddm script is removed.

**Comment by mrunixx** [comment, marked as answer]
Create the file called sddmwall.sh, past the contents below in there.
You can run this script using bash sddmwall.sh:

```
#! /bin/bash
sddmback="/usr/share/sddm/themes/Corners/backgrounds/bg.png"
sddmconf="/usr/share/sddm/themes/Corners/theme.conf"
slnkwall="${XDG_CONFIG_HOME:-$HOME/.config}/swww/wall.set"

if [ "$(getfacl -p /home/${USER} | grep user:sddm | awk '{print substr($0,length)}')" != "x" ] ; then
    echo "granting sddm execution access to /home/${USER}..."
    setfacl -m u:sddm:x /home/${USER}
fi

echo "Copying dynamic wallpaper to SDDM background location..."
cp "${slnkwall}" "${sddmback}"

echo "setting static sddm background..."
sed -i "/^Background=/c\Background=\"${sddmback}\"" "${sddmconf}"
```

**Reply by AfthabEK** [reply]
Thank you!! It worked. Why is the script removed from the Scripts?

**Reply by mrunixx** [reply]
I am not really sure, but I found the script looking back a few commits. I think the old script wasnt really working either way, It kept failing for me but this one works.

**Comment by mahaveergurjar** [comment]
> Thank you!! It worked. Why is the script removed from the Scripts?

I think this script removed because  no longer need this script every time the wallpaper in the corners sddm theme sets the same wallpaper that is on Hyprland 
```
Background="/home/mahaveer/.config/swww/wall.set"
``` 

![240325_20h31m22s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/5df7709b-4788-492f-b223-18f146db1d7d)
